tests/plugins/test_authentication.py

```diff
@@ -30,7 +30,7 @@ def test_allow_anonymous(self):
         s.username = ""
         auth_plugin = AnonymousAuthPlugin(context)
         ret = self.loop.run_until_complete(auth_plugin.authenticate(session=s))
-        self.assertTrue(ret)
+        assert ret
```

Comment:
Wouldn't it be better to `assert is True`? What if `auth_plugin.authenticate` changes with regard to the returned value?
Same for lines 61 and 81.

Reply:
I prefer to not spread the typing information all over the tests but if we want to be strict in regards to types we should add typing hints to the corresponding function, e.g. `-> bool` to `authenticate` and include a type checker in our tests

Reply:
Sounds good. Type hinting and `mypy` should be the way to go here.